akka-sample-cluster-client-grpc-java/pom.xml

```diff
@@ -1,99 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <akka.version>2.6.20</akka.version>
-    <akka.grpc.version>0.8.4</akka.grpc.version>
```

Comment:
I think that the issue here is that this samples repository was using Akka GRPC at version 0.8.4 (april 2020). Akka GRPC is now at 2.3.0
https://github.com/akka/akka-grpc/tags?after=v1.1.0

Our pekko-grpc is forked from akka-grpc last committed to in september 2022 -- Akka gRPC 2.1.6 --https://github.com/apache/incubator-pekko-grpc/commits/forked-from-akka 
https://github.com/akka/akka-grpc/tree/v2.1.6

So obviously between these two versions there have been breaking changes in the way that ServiceClients (?) are handled, which we have got to integrate into the samples library now.